fwoa-tools/src/migrationVerify.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import { readFileSync, writeFileSync } from 'fs';
+import { HealthLake, S3 } from 'aws-sdk';
+import { aws4Interceptor } from 'aws4-axios';
+import axios from 'axios';
+import * as dotenv from 'dotenv';
+import yargs from 'yargs';
+import { ExportOutput, getFhirClient, getFhirClientSMART, verifyResource } from './migrationUtils';
+
+dotenv.config({ path: '.env' });
+const { DATASTORE_ID, DATASTORE_ENDPOINT, API_AWS_REGION, IMPORT_OUTPUT_S3_BUCKET_NAME } = process.env;
+
+const IMPORT_VERIFICATION_LOG_FILE_PREFIX: string = 'import_verification_';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function parseCmdOptions(): any {
+  return yargs(process.argv.slice(2))
+    .usage('Usage: $0 [--smart, -s boolean] [--dryRun, -d boolean]')
+    .describe('smart', 'Whether the FWoA deployment is SMART-on-FHIR or not')
+    .boolean('smart')
+    .default('smart', false)
+    .alias('s', 'smart')
+    .describe('dryRun', 'Check operations and authentication status')
+    .boolean('dryRun')
+    .default('dryRun', false)
+    .alias('d', 'dryRun').argv;
+}
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const argv: any = parseCmdOptions();
+const smartClient: boolean = argv.smart;
+const dryRun: boolean = argv.dryRun;
+const logs: string[] = [];
```

Comment:
Can we update this to be a WriteStream instead? See migrationExport in develop for an example